src/js/components/Meter/stories/Circle.js

```diff
@@ -22,11 +22,7 @@ export const Circle = () => {
   return (
     <Grommet theme={grommet}>
       <Box align="center" pad="large">
-        <Meter
-          type="circle"
-          background="light-2"
-          values={[{ value, color: value > 50 ? 'accent-2' : 'accent-1' }]}
-        />
+        <Meter type="semicircle" background="light-2" value={90} max={100} />
```

Comment:
I don't think we should be removing the "circle" type here. I think we should have a separate story?

Reply:
yes I had it only for testing purposes but I  added `semicircle` story then I can 
> *  merge Circle and Bar to have a single story for Type

Reply:
Think we still need to remove "type=semicircle" from here